packages/react/README.md

```diff
@@ -1,6 +1,6 @@
 # carbon-components-react
 
-> A collection of [Carbon Components](https://github.com/IBM/carbon-components)
+> A collection of [Carbon Components](https://github.com/carbon-design-system/carbon)
```

Comment:
Or maybe this link should be `https://github.com/carbon-design-system/carbon/tree/master/packages/components` instead? 🤔

Reply:
Yea i think direct link to the components package is better